README.md

```diff
@@ -11,7 +11,7 @@
 [![Sponsors on Open Collective](https://opencollective.com/swag/sponsors/badge.svg)](#sponsors) [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Fswaggo%2Fswag.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Fswaggo%2Fswag?ref=badge_shield)
 
 
-Swag converts Go annotations to Swagger Documentation 2.0. We've created a variety of plugins for popular [Go web frameworks](#supported-web-frameworks). This allows you to quickly integrate with an existing Go project (using Swagger UI).
+Swag converts Go annotations to Swagger Documentation 2.0. It requires Go 1.9 or newer to compile. We've created a variety of plugins for popular [Go web frameworks](#supported-web-frameworks). This allows you to quickly integrate with an existing Go project (using Swagger UI).
```

Comment:
Because we also have provide  pre-compiled binaries. Would you please move Go version requirements to https://github.com/swaggo/swag/edit/master/README.md#L41 with the following like:
```
Download swag by using:
...
To build from source you need Go (1.9 or newer).
```